### PR TITLE
chore: Removing UWP

### PR DESCRIPTION
**Pull request by nickrandolph**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md). (for bug fixes / features) 
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: build/ci/.azure-pipelines.yml [hunk 1]
```diff
@@ -71,5 +66,5 @@ stages:
   # Don't trigger this stage if only docs files are changed
   condition: ne(dependencies.Determine_Changes.outputs['evaluate_changes.DetermineChanges.docsOnly'], 'true')
   jobs:
-  - template: stage-build-uitests-wasm.yml
-  - template: stage-build-runtimetests-skia.yml
```
Comment (dansiegel) [inline build/ci/.azure-pipelines.yml:75]: Can we put a note on why this is commented out.

File: samples/MauiEmbedding/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,167 @@
+; This file is for unifying the coding style for different editors and IDEs.
```
Comment (dansiegel) [inline samples/MauiEmbedding/.editorconfig:1]: I would probably avoid having more than one .editorconfig in the repo

File: samples/MauiEmbedding/MauiEmbedding/App.xaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      <ResourceDictionary.MergedDictionaries>
+<!-- Load WinUI resources -->
+		<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
+		<MaterialToolkitTheme xmlns="using:Uno.Toolkit.UI.Material"
+			ColorOverrideSource="ms-appx:///Styles/ColorPaletteOverride.xaml"
+			FontOverrideSource="ms-appx:///Styles/MaterialFontsOverride.xaml" />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </Application.Resources>
+  <!-- Add resources here -->
```
Comment (dansiegel) [inline samples/MauiEmbedding/MauiEmbedding/App.xaml:17]: Can you move/delete this comment

File: samples/MauiEmbedding/MauiEmbedding/App.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                    // TODO: Register your services
+                    //services.AddSingleton<IMyService, MyService>();
+                })
+                .UseNavigation(RegisterRoutes)
+            );
+        MainWindow = builder.Window;
+
+        Host = await builder.NavigateAsync<Shell>();
+
+        //MainWindow.Content = new MainPage();
```
Comment (dansiegel) [inline samples/MauiEmbedding/MauiEmbedding/App.xaml.cs:78]:
```suggestion
```

File: samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj [hunk 5]
```diff
@@ -1,69 +1,51 @@
...
+    <!--
+      UnoFeatures let's you quickly add and manage implicit package references based on the features you want to use.
+      https://aka.platform.uno/singleproject-features
+    -->
+    <UnoFeatures>
+      Material;
+      Dsp;
+      Toolkit;
+      Mvvm;
+      ThemeService;
```
Comment (dansiegel) [inline samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj:37]: Wait doesn't this bring in the Extensions CoreUI Package?

File: samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj [hunk 6]
```diff
@@ -1,69 +1,51 @@
...
-				<Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
-				<Compile Update="**\*.xaml.cs">
-					<DependentUpon>%(Filename)</DependentUpon>
-				</Compile>
-				<PRIResource Include="**\*.resw" />
-			</ItemGroup>
-		</Otherwise>
-	</Choose>
+  <ItemGroup>
+    <PackageReference Include="Uno.Core.Extensions.Logging.Singleton"/>
```
Comment (dansiegel) [inline samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj:42]: Isn't this brought in for you by the SDK?

File: samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj [hunk 7]
```diff
@@ -1,69 +1,51 @@
...
-		<UnoImage Include="Assets\**\*.svg" />
-		<EmbeddedResource Include="appsettings.json" />
-		<EmbeddedResource Include="appsettings.*.json" DependentUpon="appsettings.json" />
-		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
-	</ItemGroup>
 
 	<ItemGroup>
-		<ProjectReference Include="..\..\..\src\Uno.Extensions.Configuration\Uno.Extensions.Configuration.csproj" />
+    <ProjectReference Include="..\..\..\src\Uno.Extensions.Core.UI\Uno.Extensions.Core.WinUI.csproj" />
+    <ProjectReference Include="..\..\..\src\Uno.Extensions.Configuration\Uno.Extensions.Configuration.csproj" />
```
Comment (dansiegel) [inline samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj:48]: Looks like you need to normalize the line endings between tabs/spaces

File: samples/MauiEmbedding/MauiEmbedding/Package.appxmanifest [hunk 8]
```diff
@@ -7,19 +7,18 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="5E560C41-9B2D-4E0D-85AD-F88AC0002C15"
-    Publisher="O=TestHarness"
+    Name="MauiEmbedding"
+    Publisher="O=NickRandolph"
```
Comment (dansiegel) [inline samples/MauiEmbedding/MauiEmbedding/Package.appxmanifest:11]:
```suggestion
    Publisher="O=Uno Platform"
```

Comment (dansiegel) [reply samples/MauiEmbedding/MauiEmbedding/Package.appxmanifest:11]: Actually can you check with a new project... this seems to have a bunch of nodes that have been removed and should be provided by Resizetizer now.

File: samples/Playground/Playground/App.xaml [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+                              Margin="0,0,0,50" />
+              </Grid>
+            </DataTemplate>
+          </Setter.Value>
+        </Setter>
+      </Style>
+    </ResourceDictionary>
+  </Application.Resources>
+
+  <!-- Add resources here -->
```
Comment (dansiegel) [inline samples/Playground/Playground/App.xaml:38]: can remove/move this

**Review by dansiegel — APPROVED** [review]

File: samples/MauiEmbedding/.editorconfig (hunk 2 above)
Comment (nickrandolph) [reply samples/MauiEmbedding/.editorconfig]: Done

File: samples/MauiEmbedding/MauiEmbedding/App.xaml (hunk 3 above)
Comment (nickrandolph) [reply samples/MauiEmbedding/MauiEmbedding/App.xaml]: Done

File: samples/MauiEmbedding/MauiEmbedding/App.xaml.cs (hunk 4 above)
Comment (nickrandolph) [reply samples/MauiEmbedding/MauiEmbedding/App.xaml.cs:78]: done

File: samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj (hunk 5 above)
Comment (nickrandolph) [reply samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj:37]: Removed ThemeService

File: samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj (hunk 6 above)
Comment (nickrandolph) [reply samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj:42]: doesn't seem to be

File: samples/Playground/Playground/App.xaml (hunk 9 above)
Comment (nickrandolph) [reply samples/Playground/Playground/App.xaml]: done

**Pull request merged**